### DISCUSSION TITLE
Uniqueness of regions name

### DISCUSSION THREAD

**Discussion by Kani999** [netbox-community/netbox#7483] [Ideas]
Hello,

We are using Regions in netbox with the nested recursion to construct a hierarchy as the documentation proposes.
For the source of data, we use a national database of regions, districts, and cities. 
We are running into a problem if there is the same name of the city in the different countries.

For Example, the `Springfield` cannot exist multiple times, because it violates the uniqueness of the `Name`.
```
- Massachusetts
   - Springfield  
- Missouri
   - Springfield  
- Illinois 
   - Springfield  
- Ohio
   - Springfield   
- Oregon
   - Springfield 
```

It would make more sense to us if the validation of the name would not be violated if there is a different parent assigned to them in the hierarchy. Otherwise, there always has to be some "difference" in the name of the city. And because the source of the data becomes from the national database, there is no way how to clearly distinguish them.

**Comment by jeremystretch** [comment]
I think it would make sense to relax this constraint so that the name of a region (or site group) must be unique only within its assigned parent (if any).

**Reply by Kani999** [reply]
Should I create an issue about this? If it's a proper way how to escalate it.

**Reply by jeremystretch** [reply]
I'd say for now, maybe comment on #7354 with the suggestion to expand its scope to include regions as well. It seems like a very similar concern. There are probably others models we should revisit as well.
